knowledge-base/kb-richtextbox-auto-text-implementation.md

```diff
@@ -0,0 +1,92 @@
+---
+title: RadRichTextBox AutoText implementation
+description: Shows how you can create reusable text snippets.
+type: how-to
+page_title: RadRichTextBox AutoText implementation
```

Comment:
it is nice to use a different value for page_title as it would be beneficial in terms of SEO